### ISSUE TITLE
Paired over TCP, but client says "No bluetooth server"

### ISSUE THREAD

**Issue by petko10** [rootkiwi/an2linuxclient#23]
The app is not supposed to require a bluetooth server, right ?
I just installed from AUR ( r6.4c03e19-1 ) and Android (0.8.0) , and can't seem to receive a test notification. When I click on display test notification I get the "Found no enabled Bluetooth server". I guess I could be missing something, but I have no idea what atm.

**Comment by rootkiwi** [comment]
Yes that is rather confusing. It does not require a bluetooth server, it just notifies about that because bluetooth is turned on.

Why you dont recv a test notification I dont know

**Comment by petko10** [comment]
Thanks for the reply! Yes, the problem is that bluetooth was turned on, but I'd consider that a bug, since I'm using bluetooth for other stuff, and that shouldn't interfere with the wifi an2linux server. I'm still not getting a notification on the desktop when I click "Display test..." . Does an2linux display notifications via libnotify or some other way?

**Comment by rootkiwi** [comment]
Yeah that may be considered a bug yes. It uses libnotify.
